util/importers/ldc93s1.py

```diff
@@ -74,14 +80,14 @@ def _compute_source_target(self):
 
     def _populate_batch_queue(self, session):
         source, source_len, target, target_len = self._compute_source_target()
-        while True:
+        while not self._coord.should_stop():
             try:
                 session.run(self._enqueue_op, feed_dict={
                     self._x: source,
                     self._x_length: source_len,
                     self._y: target,
                     self._y_length: target_len})
-            except (RuntimeError, tf.errors.CancelledError):
+            except Exception as e:
```

Comment:
Why is this required? I once tried to make it as specific as possible. Now it captures everything and by that it'd make debugging much harder.